readme.md

```diff
@@ -48,7 +48,7 @@ Milliseconds to humanize.
 
 #### options
 
-##### secDecimalDigits
+##### sec-decimal-digits
```

Comment:
Do you mean short tags as in `-v` and `-c` ?

So, `pretty-ms -v` would be equivalent to `pretty-ms --verbose` ?